### ISSUE TITLE
Make more clear registration is required

### ISSUE THREAD

**Issue by Elmi77** [DaylightingSociety/WhereAreTheEyes#23]
I used my app for the first time yesterday and registered about 10 cameras. AFTER that I got a message telling me I have to register my username at the webpage - resulting in a loss of these 10 positions.

So the App should be more clear about this or should store and upload such positions after successful registration or should force registration on first start-up.

**Comment by milo-trujillo** [comment]
That should not be possible. When you try to mark the first camera the app should immediately prompt you to set a username. Were you offline or in odd network conditions when you marked the ten cameras? Could you also tell me whether you are on iOS or Android? I have not seen this behavior.

I agree, registration should be made more clear. It is not mandatory because some users only want to browse the map, for which no username is needed.

**Comment by Elmi77** [comment]
I have been online via mobile connection (quality of this connection is not clear but should not have been that bad). It took at least 1 minute before the message popped up, my phone's screen was switched of in the meantime, so I have seen the message about 5 minutes later. It is an Android 4.4.2 phone

**Comment by milo-trujillo** [comment]
Hmm, that's odd. What was the exact message you got? It should pop up an error right away if you have no username set in preferences.

If you *do* have a username set, but it's not a registered one, then the phone should vibrate and send off a network request, and an error message will appear when the server returns that the username is wrong.

**Comment by Elmi77** [comment]
Yes, it was exactly that message with the "no username" which then guided me to the webpage where I could register. Anyway, a clear message at start-up of the application that informs about the need for registration should solve this problem. This message could be shown as long as the user is not registered.

**Comment by milo-trujillo** [comment]
> A clear message at start-up of the application that informs about the need for registration should solve this problem

I agree. Adding an alert at start time if no username is set is now planned for the next release.

**Comment by milo-trujillo** [comment]
This is now implemented on iOS for v1.4. Next up, implementing on Android.
